content/docs/develop/reference/bridge/basic_authentication/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Basic Authentication
+description: Enable/Disable basic authentication
+weight: 21
+keywords: [bridge]
+---
+
+The Keptn Bridge has a basic authentication feature, which can be controlled by setting the following two environment variables:
```

Comment:
Please add "setting the following two environment variables in the deployment of the bridge:"

Reply:
I added the info.